/**:
    ros__parameters:
      hmax_blue: 115
      hmax_purple: 140
      hmax_red: 10
      hmin_blue: 105
      hmin_purple: 125
      hmin_red: 0
      use_sim_time: false
      x_max: 79
      x_min: 29
      y_max: 46
      y_min: 0
